### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]:Keyboard focus is not visible for "Next Page" button which is present in ACR Explorer window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1505]
**Repro Steps:**

1.  Open and Start Eclipse.
2. Create a new dynamic web project.
3. In "Azure Explorer" , select the pushed image under container registries and press shift+F10 to open the context menu.
4. Navigate to and select "Open ACR Explorer" menu item
5. The open window , navigate to list items and press tab then observe the keyboard focus for "Next Page" button

**Expected Result:**
Keyboard focus is not visible for "Next Page" button 

**Actual Result:**
Keyboard focus should be visible for "next Page" button
Observation: Keyboard focus is not visible but by hitting enter on "Next Page" button we are able to access. Once user traverse the "Next Page"  we are able to see the focus for the next time navigation for both "previous Page" and "Next page" button.

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)

![keyboard focus is not visible in acr explorer](https://user-images.githubusercontent.com/22956130/36142966-acd1efe0-10cf-11e8-9244-7f18bdbeaf5e.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by jdneo** [comment]
@WiproPSAccTest The `next page` and `previous page` are items of the toolbar, to navigate in toolbar, you can use `up`, `down`, `left`, `right`.

**Comment by WiproPSAccTest** [comment]
Verified the issue in V 3.6.0.
Issue got fixed

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749937
